### PR TITLE
Create PS Q3 feedback report

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/307489194

Changes proposed in this pull request:

- Create PS Q3 feedback report
- 
-

### CODE REVIEW

**Comment by nancymic2** [comment]
Still checking formatting, but before it gets too late, requesting tech writer approval as this needs to be published today. Thanks!

**Review by kevinkiklee — CHANGES_REQUESTED** [review]
Looks like the content hierarchy is a little off, and needs to be updated. 

This is the Q3 report that was added: 
image

This is the previous Q2 report: 
image

File: site/en/docs/privacy-sandbox/feedback/report-2023-q2.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ subhead: >
   Quarterly report for 2023 Q2 summarising the ecosystem feedback received on
   Privacy Sandbox proposals and Chrome's response.
 description: >
-  Quarterly report for 2023 Q1 summarising the ecosystem feedback received on
+  Quarterly report for 2023 Q2 summarising the ecosystem feedback received on
```
Comment (samdutton) [inline site/en/docs/privacy-sandbox/feedback/report-2023-q2.md:8]: "summarizing" (for US English)?

**Comment by nancymic2** [comment]
Thanks Kevin. The converter makes h2s into h1s etc. this would cause a build error as well!

File: site/en/docs/privacy-sandbox/feedback/report-2023-q2.md (hunk 1 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/feedback/report-2023-q2.md:8]: I've always left that because it's CMA. Should I change?

Comment (nancymic2) [reply site/en/docs/privacy-sandbox/feedback/report-2023-q2.md:8]: it's been that way for a few years

Comment (samdutton) [reply site/en/docs/privacy-sandbox/feedback/report-2023-q2.md:8]: Ah — sorry. Forgot that this is CMA language.

Comment (nancymic2) [reply site/en/docs/privacy-sandbox/feedback/report-2023-q2.md]: well, now I don't know :). I just changed them all :)

File: site/en/docs/privacy-sandbox/feedback/report-2023-q3.md [hunk 2]
```diff
@@ -0,0 +1,1758 @@
...
+  </thead>
+  <tbody>
+    <tr>
+      <td style="vertical-align: top;">Cross-device</td>
+      <td style="vertical-align: top;">Plan for cross-device support for Attribution Reporting API.</td>
+      <td style="vertical-align: top;">Cross-device presents new privacy challenges on top of 3PC and also
+adds technology distribution challenges given the range of devices
+and platforms a user might use. We are exploring potential solutions,
+but we are focused on the critical use cases currently supported by
+ARttribution Reporting and do not have plans to introduce cross-device support before
```
Comment (samdutton) [inline site/en/docs/privacy-sandbox/feedback/report-2023-q3.md:1120]: Attribution

File: site/en/docs/privacy-sandbox/feedback/report-2023-q3.md [hunk 3]
```diff
@@ -0,0 +1,1758 @@
...
+      <td style="vertical-align: top;">How are user interactions tracked?</td>
+      <td style="vertical-align: top;">Bounce tracking mitigations track two types of user
+interactions:<br>
+<br>
+<ul>
+<li>User activations as defined by the <a
+href="https://html.spec.whatwg.org/#tracking-user-activation">html
+spec</a>. These are basically clicks, key presses, touch screen taps, etc.</li>
+
+<li>Successful <a href="https://w3c.github.io/webauthn/">webauthn</a>
```
Comment (samdutton) [inline site/en/docs/privacy-sandbox/feedback/report-2023-q3.md:1521]: WebAuthn 

(link text)

**Review by samdutton — APPROVED** [review]
LGTM with a couple of tweaks.

File: site/en/docs/privacy-sandbox/feedback/report-2023-q3.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/feedback/report-2023-q3.md]: done

File: site/en/docs/privacy-sandbox/feedback/report-2023-q3.md (hunk 3 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/feedback/report-2023-q3.md]: done

**Review by rachelandrew — APPROVED** [review]

**Review by kevinkiklee — APPROVED** [review]

**Pull request merged**